components/raftstore/src/store/config.rs

```diff
@@ -66,6 +66,8 @@ pub struct Config {
     // When the approximate size of raft log entries exceed this value,
     // gc will be forced trigger.
     pub raft_log_gc_size_limit: ReadableSize,
+    // Old logs in Raft engine needs to be purged peridically.
+    pub raft_engine_purge_interval: ReadableDuration,
```

Comment:
There has been a configure `raft_log_gc_tick_interval` used to check whether the log is too large. Could we use it to check old log of `raft-engine`?

Reply:
They are different. `raft_log_gc_tick_interval` is for one region but the latter is for whole engine.